docs/API_DESIGN.md

```diff
@@ -606,33 +607,16 @@ CLASS DocumentMetadata
 
 ```pseudocode!
 CLASS DidJwk
-  PUBLIC DATA did: Did
-  PUBLIC DATA document: Document
-  CONSTRUCTOR(public_jwk: Jwk)
-  CONSTRUCTOR(uri: string)
+  STATIC METHOD create(options: DidJwkCreateOptions): BearerDid
   STATIC METHOD resolve(uri: string): ResolutionResult
 ```
 
-#### Example: Create a `did:jwk`
-
-```pseudocode!
-key_manager = new InMemoryKeyManager()
-public_jwk = key_manager.import_private_jwk(Ed25519Generator::generate())
-did_jwk = new DidJwk(public_jwk)
-```
-
-#### Example: Instantiate an existing `did:jwk`
-
-```pseudocode!
-uri = "did:jwk:eyJrdHkiOiJFQyIsInVzZSI6InNpZyIsImNydiI6InNlY3AyNTZrMSIsImtpZCI6ImkzU1BSQnRKS292SEZzQmFxTTkydGk2eFFDSkxYM0U3WUNld2lIVjJDU2ciLCJ4IjoidmRyYnoyRU96dmJMRFZfLWtMNGVKdDdWS04VEZaTm1BOVlnV3p2aGg3VSIsInkiOiJWTEZxUU1aUF9Bc3B1Y1hvV1gyLWJHWHBBTzFmUTVMbjE5VjVSQXhyZZVIiwiYWxnIjoiRVMyNTZLIn0"
-did_jwk = new DidJwk(uri)
-```
-
-#### Example: Resolve a `did:jwk`
+#### `DidJwkCreateOptions`
 
 ```pseudocode!
-uri = "did:jwk:eyJrdHkiOiJFQyIsInVzZSI6InNpZyIsImNydiI6InNlY3AyNTZrMSIsImtpZCI6ImkzU1BSQnRKS292SEZzQmFxTTkydGk2eFFDSkxYM0U3WUNld2lIVjJDU2ciLCJ4IjoidmRyYnoyRU96dmJMRFZfLWtMNGVKdDdWS04VEZaTm1BOVlnV3p2aGg3VSIsInkiOiJWTEZxUU1aUF9Bc3B1Y1hvV1gyLWJHWHBBTzFmUTVMbjE5VjVSQXhyZZVIiwiYWxnIjoiRVMyNTZLIn0"
-resolution_result = DidJwk.resolve(uri)
+CLASS DidJwkCreateOptions
+  PUBLIC DATA key_manager: KeyManager?
+  PUBLIC DATA dsa: Dsa?
```

Comment:
the dsa in this readme needs the new algs I think

Reply:
done